### ISSUE TITLE
About autocast

### ISSUE THREAD

**Issue by ziyaxuanyi** [pytorch/pytorch#86548]
Can i use autocast in model inference? Is that can accelerate the inference time?

**Comment by sanchitintel** [comment]
On what device?

**Comment by ziyaxuanyi** [comment]
NVIDIA TESLA T4

**Comment by ziyaxuanyi** [comment]
> On which device?

NVIDIA TESLA T4

**Comment by sanchitintel** [comment]
Not an official answer, but you might want to check out https://pytorch.org/tutorials/recipes/recipes/amp_recipe.html. Thanks!

**Comment by ptrblck** [comment]
Yes, you can use `autocast` during inference but could also try to call `.half()` on the model and data directly.
